frontend/public/components/about-modal.tsx

```diff
@@ -21,32 +22,35 @@ const AboutModalItems: React.FC<AboutModalItemsProps> = ({cv}) => {
   const channel: string = _.get(cv, 'data.spec.channel');
   const lastUpdate: UpdateHistory = _.get(cv, 'data.status.history[0]');
   return (
-    <TextContent>
-      <TextList component="dl">
-        {lastUpdate && (
-          <React.Fragment>
-            <TextListItem component="dt">OpenShift Version</TextListItem>
-            <TextListItem component="dd">{lastUpdate.state === 'Partial' ? `Updating to ${lastUpdate.version}` : lastUpdate.version}</TextListItem>
-          </React.Fragment>
-        )}
-        <TextListItem component="dt">Kubernetes Version</TextListItem>
-        <TextListItem component="dd">{kubernetesVersion}</TextListItem>
-        {channel && (
-          <React.Fragment>
-            <TextListItem component="dt">Channel</TextListItem>
-            <TextListItem component="dd">{channel}</TextListItem>
-          </React.Fragment>
-        )}
-        {clusterID && (
-          <React.Fragment>
-            <TextListItem component="dt">Cluster ID</TextListItem>
-            <TextListItem component="dd">{clusterID}</TextListItem>
-          </React.Fragment>
-        )}
-        <TextListItem component="dt">API Server</TextListItem>
-        <TextListItem component="dd">{window.SERVER_FLAGS.kubeAPIServerURL}</TextListItem>
-      </TextList>
-    </TextContent>
+    <React.Fragment>
+      {hasAvailableUpdates(cv.data) && <Alert className="co-about-modal__alert" variant="info" title={<React.Fragment>Update available. <a href="/settings/cluster">View Cluster Settings</a></React.Fragment>} />}
```

Comment:
Use `Link` not `a` to avoid reloading the entire console

Reply:
Whoops, sorry. Updated the commit with the change.